internal/runners/branch/list.go

```diff
@@ -31,13 +34,22 @@ func (l *List) Run() error {
 
 	project, err := model.FetchProjectByName(l.project.Owner(), l.project.Name())
 	if err != nil {
-		return locale.WrapError(err, "branch_list_project_err", "Could not get details for project: {{.V0}}", l.project.Namespace().String())
+		return locale.WrapError(err, "err_fetch_project", l.project.Namespace().String())
```

Comment:
```suggestion
		return locale.WrapError(err, "err_fetch_project", "", l.project.Namespace().String())
```

Third arg is the fallback string.